client/dao/DogLicensesDao.js

```diff
@@ -4,7 +4,10 @@ import type { LoopbackGraphql } from '../loopback-graphql';
 import type { DogLicense } from '../types';
 
 import searchDogLicenses from '../queries/search-dog-licenses';
+import mutateDogLicenses from '../queries/mutate-dog-licenses';
```

Comment:
rename `mutate` to `submit` or `create` or something like that, here and elsewhere.